src/handlers/commands_fixed.py

```
@@ -0,0 +1,224 @@
+            await delete_user_message(m)
+
+def register_callback_handlers(dp, pool, bot_instance):
+    """Регистрирует обработчики callback'ов"""
+    
+    @dp.callback_query(F.data == "back_to_main")
+    async def cb_back(c: CallbackQuery, state: FSMContext):
+        await state.clear()
+        await delete_all_bot_messages(c.from_user.id, bot_instance)
+        try:
+            msg = await c.message.edit_text(T["start"], reply_markup=kb_main())
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)